command.go

```diff
@@ -1018,7 +1018,8 @@ func (c *Command) ValidateArgs(args []string) error {
 	return c.Args(c, args)
 }
 
-func (c *Command) validateRequiredFlags() error {
+// ValidateRequiredFlags validates the required flags logic and returns all unset required flags.
```

Comment:
Sorry to be picky but can you chance the godoc to:

// ValidateRequiredFlags validates all required flags are present and returns an error otherwise

Reply:
No, it's not picky at all. It's very professional comments.  😄